.gitignore

```diff
@@ -15,7 +15,6 @@ node_modules
 /src
```

Comment:
You should create a branch from master so you can merge it on master

Reply:
Ok, I will close this pull request then